frontend/javascripts/oxalis/view/left-border-tabs/mapping_settings_view.tsx

```diff
@@ -113,7 +113,10 @@ class MappingSettingsView extends React.Component<Props, State> {
       return;
     }
 
-    if (this.props.segmentationLayer?.mappings != null) {
+    if (
+      this.props.segmentationLayer?.mappings != null &&
+      this.props.segmentationLayer?.agglomerates != null
+    ) {
       return;
     }
```

Comment:
Both good points :+1 I renamed the method to `ensureMappingsAreLoaded` since the layer is a prop of the component and doesn't need to be mentioned in the method names of the component imo :)